### PR TITLE
Newsweek clone site

### PR DESCRIPTION
**Pull request by sinansevgi**
We used Bootstrap framework to create replica of the Newsweek.com website.

-Bootstrap v4.5.2 used on the project.
-We tried to use bootstrap classes wherever possible.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+.object-fit-cover {
+  object-fit: cover;
+}
+
+.text-red {
+  color: var(--newsweek-red);
+}
+
+.text-dark {
+  color: #000 !important;
```
Comment (umairarshadbutt) [inline css/style.css]: Remove `!important`

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+}
+
+.text-blunt {
+  color: #494f54 !important;
```
Comment (umairarshadbutt) [inline css/style.css]: Remove `!important`

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+.fs-22 {
+  font-size: 22px;
+}
+
+.fs-24 {
+  font-size: 24px;
+}
+
+.border-thick {
+  border-width: 4px !important;
```
Comment (umairarshadbutt) [inline css/style.css]: Remove `!important`

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi 
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [x] Remove `!important `from css file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)
- [x] Remove horizontal scrollbar(I'm attaching the [video](https://www.awesomescreenshot.com/video/1293309?key=f99eff64e084a95241c790d05231f407))
**Main Page**
- [x] Text should overflow on images in medium and small screens

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://sinansevgi.github.io/newsweek-clone/)|
|:---:|:---:|
|source-page|clone|

**Footer**
- [x] Design responsive footer for medium and small screens

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://sinansevgi.github.io/newsweek-clone/)|
|:---:|:---:|
|source-page|clone|

- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by Rhelli — APPROVED** [review]
# Approved! 🚀✅

Well done on implementing the changes suggested by the previous TSE. You've managed to make an exceptionally responsive website that stays true to Newsweek's own website. I have a couple of suggestions that you could implement or carry through to your next project to take your work to the next level:
 - **[Optional]** - Make sure you use the `footer` tag on your webpage footers in the same way you've used the `nav` tag. This keeps your HTML semantic and accessible.
 - **[Optional]** - Your search bar magnifying glass icon is broken on full screen due to having a property of `right: 0;`. If you remove this it should place it correctly.

If you have any questions, feel free to either @ me here on Giithub or get in contact with me via Slack!

All the best,

Rory 💂🏽‍♂️

**Pull request merged**